specification/synapse/data-plane/readme.python.md

```diff
@@ -4,49 +4,48 @@ These settings apply only when `--python` is specified on the command line.
 Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
 Use `--python-mode=update` if you already have a setup.py and just want to update the code itself.
 
-``` yaml
+``` yaml $(python)
```

Comment:
You don't need to add that, as you're already in the readme.python.md, so it has been filtered already to python. IOW, this doesn't help as `$(python)` will always be true anyway.